config/src/config.rs

```diff
@@ -1491,6 +1499,15 @@ impl DroppedFileQuoting {
             Self::SpacesOnly => s.replace(" ", "\\ "),
             // https://docs.rs/shlex/latest/shlex/fn.quote.html
             Self::Posix => shlex::quote(s).into_owned().to_string(),
+            Self::Windows => {
+                let chars_need_quoting = [' ','\t','\n','\x0b','\"'];
+                if chars_need_quoting.iter().any(|&c| s.contains(c)) {
```

Comment:
I would flip this around to be something like this, so that there is a single pass over what is likely to be the longer string; the overlay big-O should be lower when phrased this way:

```rust
s.chars().any(|c| chars_need_quoting.contains(c))
```

Reply:
Yes, this is better in performance.